CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+* **Data List**: Changes to the breakpoint at which the Data List padding changes for a mobile view to align with the padding change in the Page component. The mobile breakpoint used to be defined as $pf-data-list--breakpoint--mobile with a custom value of 380px using custom properties with --mobile in the name. The breakpoint is now defined using our existing global breakpoint variable $pf-global--breakpoint--md with --md in the custom property names. ([#1651](https://github.com/patternfly/patternfly-next/issues/1651))
+* **Data List**: Removed the use of $pf-data-list--breakpoint--mobile and replaces it with our standard global breakpoint variables. Any use of $pf-data-list--breakpoint--mobile should be updated to use the standard breakpoint variables. Any custom properties that had --mobile in the name referencing the old breakpoint should update to the new custom property name that uses our standard breakpoint size in the name. I.e, --sm, --md, --lg, etc. ([#1658](https://github.com/patternfly/patternfly-next/issues/1658))
+* **Data List**: We refactored the data list HTML and CSS. The basic HTML structure of the data list has changed to support control (.pf-c-data-list__item-control), content (.pf-c-data-list__item-content), and action (.pf-c-data-list__item-action) columns, with rows (.pf-c-data-list__item-row), defined in the content column. The previous version of the data list HTML will no longer work and needs to be refactored. The same applies for the CSS. ([#1695](https://github.com/patternfly/patternfly-next/issues/1695))
+* **Data List**: The mobile breakpoint and variable names for the left/right padding in Data List to match the Page component. The mobile breakpoint changed from $pf-global--breakpoint--sm to $pf-global--breakpoint--sm and the variable names at this breakpoint now use --md in the name instead of --sm. ([#1660](https://github.com/patternfly/patternfly-next/issues/1660))
+* **Empty State**: The default width of the Empty State component was changed to be full-width by default. If you want to change the width, you now use the .pf-m-sm and .pf-m-lg modifiers. The previous default width was the equivalent of .pf-m-lg. ([#1475](https://github.com/patternfly/patternfly-next/issues/1475))
+* **Nav**: Divider used to separate links in vertical navigation will break.The divider was previously defined as a modifier on a link (`<a>`) in the vertical navigation. The divider is now it’s own list item (`<li>`) element defined as .pf-c-nav__separator. ([#1586](https://github.com/patternfly/patternfly-next/issues/1586))
```

Comment:
Should we remove the backticks here? Doesn't look like we're formatting tags anywhere else in the doc. 

```suggestion
* **Nav**: Divider used to separate links in vertical navigation will break.The divider was previously defined as a modifier on a link (<a>) in the vertical navigation. The divider is now it’s own list item (<li>) element defined as .pf-c-nav__separator. ([#1586](https://github.com/patternfly/patternfly-next/issues/1586))
```

Reply:
We have to leave them - or the markdown will blow up